DataDog/dd-agent
# [ceph] add metric for active monitors #2719

### What does this PR do?

This PR adds code to create the ceph.num_mons.active metric.  This metric will return the number of _active_ mon processes as seen in the number of quorum members.  This differs from the existing ceph.num_mons metric, which returns the number of _configured_ mon processes.
### Motivation

In Ceph, a quorum of mon processes in the cluster are required.  Datadog monitors cannot alert to dangers unless the number of active mons (compared to the number configured) is known.
### Testing Guidelines

Changes to tests add in the ceph.num_mons.active metric to check for in addition to ceph.num_mons.  No changes to test json input expected.
### Additional Notes

It's pretty simple, working fine in a test environment where we altered the base Ceph dashboard to return the number of down mon processes (ceph.num_mons - ceph.num_mons.active) and change from green to yellow if > 0, red if > 1, where the quorum of that cluster would be in danger.
